spec/factories/events.rb

```diff
@@ -30,7 +30,21 @@
       factory :event_scheduled do
         after(:build) do |event|
           event.state = 'confirmed'
-          event.event_schedules << build(:event_schedule, event: event)
+          event.event_schedules << build(:event_schedule, event: event, start_time: DateTime.current)
+        end
+      end
+
+      factory :past_event_scheduled do
+        after(:build) do |event|
+          event.state = 'confirmed'
+          event.event_schedules << build(:event_schedule, event: event, start_time: (DateTime.current - 2.days).to_s)
+        end
+      end
+      factory :future_event_scheduled do
+        after(:build) do |event|
+          event.start_time = (DateTime.current + 2.days).to_s
+          event.event_schedules << build(:event_schedule, event: event, start_time: (DateTime.current + 2.days).to_s)
```

Comment:
We don't need these factories since we moved the current scope to event_schedule.